azure/functions/authenticationevents.py

```diff
@@ -314,37 +327,64 @@ def __init__(self,
                 self.Client=Client
                 self.CorrelationId=CorrelationId
-
-class TokenIssuanceStartData(IEventData):
-    def __init__(self,
-                EventId: uuid,
-                EventTime: DateTime,
-                EventVersion: str,
-                EventType: str,
-                Context: str):
-                self.Context=Context
-                self.EventType=EventType
-                self.EventVersion=EventVersion
-                self.EventTime=EventTime
-                self.EventId=EventId
-
-
-
-class TokenIssuanceStartRequest(IEventRequest):
-    def __init__(self,
-                response: TokenIssuanceStartResponse,
-                payload: TokenIssuanceStartData,
-                TokenClaims: dict[str,str]):
-                self.TokenClaims=TokenClaims
-                self.response=response
-                self.payload=payload
-
-    def InstanceCreated(self,args: list):
-        self.TokenClaims=args[0]
+    
+    def populate(context: dict):
+        user=User.populate(context.get('user'))
+        client=Client.populate(context.get('client'))
+        authProtocol=AuthProtocol.populate(context.get('authProtocol'))
+        clientServicePrincipal=ServicePrincipal.populate(context.get('clientServicePrincipal'))
+        resourceServicePrincipal=ServicePrincipal.populate(context.get('resourceServicePrincipal'))
+        return Context(CorrelationId=context.get('correlationId'),User=user,Client=client,ClientServicePrincipal=clientServicePrincipal,ResourceServicePrincipal=resourceServicePrincipal,Roles=context.get('roles'),AuthProtocol=authProtocol)
+
+class preview_10_01_2021():
+    class TokenIssuanceStartResponse(IEventResponse):
+        def __init__(self,Description):
+                    # super().__init__(kargs)
+                    self.Description=Description
         
-                
-
+        def get_Description(self):
+            return self.Description
         
+        def set_Description(self,value):
+            self.Description=value
+
+        def invalidate(self):
+            self.set_JsonValue(["addition","description"], self.Description)
+
+    class TokenIssuanceStartData(IEventData):
+        def __init__(self,
+                    EventId: str,
+                    EventTime: DateTime,
+                    EventVersion: str,
+                    EventType: str,
+                    Context: Context):
+                    self.Context=Context
+                    self.EventType=EventType
+                    self.EventVersion=EventVersion
+                    self.EventTime=EventTime
+                    self.EventId=EventId
+
+        def CreateInstance(payload: dict):
+            context=Context.populate(payload.get('context'))
+            return preview_10_01_2021.TokenIssuanceStartData(EventId=payload.get('eventListenerId'),EventTime=payload.get('time'),EventType=payload.get('type'),EventVersion=payload.get('apiSchemaVersion'),Context=context)
```

Comment:
**payload can not be used here because in the TokenIssuanceStartData object context is an object and in **payload it is a string so therefore I created context object first which was then passed to the constructor.